Using backend: pytorch
Traceback (most recent call last):
  File "train.py", line 3, in <module>
    from parcellearning import gauss
  File "/mnt/parcellator/parcellation/Code/parcellearning/parcellearning/__init__.py", line 3, in <module>
    from . import *
  File "/mnt/parcellator/parcellation/Code/parcellearning/parcellearning/conv/__init__.py", line 1, in <module>
    from .cgatconv import CGATConv
  File "/mnt/parcellator/parcellation/Code/parcellearning/parcellearning/conv/cgatconv.py", line 8, in <module>
    import dgl
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/__init__.py", line 14, in <module>
    from .backend import load_backend, backend_name
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/backend/__init__.py", line 73, in <module>
    load_backend(get_preferred_backend())
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/backend/__init__.py", line 23, in load_backend
    mod = importlib.import_module('.%s' % mod_name, __name__)
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/backend/pytorch/__init__.py", line 1, in <module>
    from .tensor import *
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/backend/pytorch/tensor.py", line 13, in <module>
    from ...function.base import TargetCode
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/function/__init__.py", line 5, in <module>
    from .message import *
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/function/message.py", line 8, in <module>
    from .._deprecate.runtime import ir
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/_deprecate/runtime/__init__.py", line 4, in <module>
    from . import scheduler
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/_deprecate/runtime/scheduler.py", line 4, in <module>
    from ... import utils
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/utils/__init__.py", line 3, in <module>
    from .data import *
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/dgl/utils/data.py", line 4, in <module>
    import networkx as nx
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/networkx/__init__.py", line 114, in <module>
    import networkx.generators
  File "/mnt/home/keschenb/anaconda3/lib/python3.6/site-packages/networkx/generators/__init__.py", line 21, in <module>
    from networkx.generators.random_graphs import *
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 779, in get_code
  File "<frozen importlib._bootstrap_external>", line 487, in _compile_bytecode
KeyboardInterrupt
